GestaoFinanceira/GestaoFinanceira/Controllers/DashBoardController.cs

```diff
@@ -137,7 +137,7 @@ public void GenerateChart(Chart chart, ChartType typeChart, DateTime date)
                             if (cat.type == EntryType.Expense && cat.Description == entry.Category.Description)
                             {
                                 foreach (var e in listEntries)
-                                    saldoCat += e.Category.Id == cat.Id ? e.Value : 0.00; 
+                                    saldoCat += e.Category.Id == cat.Id ? e.Value : 0.00;

```

Comment:
Só uma dica

Você pode utilizar o método `.Sum(...)` pra fazer a mesma tarefa.

Neste caso não precisa do `foreach`
```suggestion                                 
                                       saldoCat = listEntries.Sum(e=>e.Category.Id == cat.Id ? e.Value : 0.00) ;
```

Reply:
Isso vale ouro!
Obrigado!